plugins/output/flight/flight.go

```diff
@@ -0,0 +1,141 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+// SampleConfig returns the configuration of the Apache Arrow Flight plugin.
+func (*Flight) SampleConfig() string {
```

Comment:
I'm a little confused here. What is this for? Why is it a *sample* rather than an actual configuration? Why do we ask the user to make configurations in a file called "sample"?

Reply:
I can see why it would be confusing, but this is the [convention](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#plugins) used in Telegraf. I believe it is because this is attached to the main telegraf.conf file once the plugin has been committed to the main Telegraf repository. It think it doesn't really matter what it is called because I clearly state in the README that this should be configured. If the user doesn't want to do that, the user can technically copy these lines into any config, modify the `command = ["/path/to/flight.exe", "-config", "/path/to/sample.conf"]` to point to that and use that instead.

Reply:
OK, I think it is a bit weird to have the actual configuration in a sample configuration. But if that is how it typically is done, then let's continue like that...